packages/core/src/Stores/client-store.js

```diff
@@ -2088,8 +2088,8 @@ export default class ClientStore extends BaseStore {
 
         if (response?.logout === 1) {
             this.cleanUp();
-
             this.root_store.rudderstack.reset();
+            this.setIsAuthorize(false);
```

Comment:
`is_authorize` is still `true` after user is logged out, had to manually set it to `false` once user logs out